src/styles/gallery.scss

```diff
@@ -0,0 +1,101 @@
+.whppt-gallery {
+  @apply bg-whpptDarkMode px-6 py-4 mb-5 flex w-full h-screen;
+
+  &__content {
+    @apply w-full flex flex-col;
+  }
+
+  &__main-container {
+    @apply flex flex-1 overflow-auto gap-1 w-full;
+    -ms-overflow-style: none;
+    scrollbar-width: none;
+
+    &::-webkit-scrollbar {
+      display: none;
+    }
```

Comment:
These are interesting. Lets chat about them. I'd like to understand if postcss will add them or could it for us or if we could make it re-usable. I'd have to understand the outcome first though.